### DISCUSSION TITLE
Should assessment-platforms be able to be included in local-definitions of POAM

### DISCUSSION THREAD

**Discussion by CyberDaedalus00** [usnistgov/OSCAL#1268] [Q&A]
In reviewing the POA&M, I noticed that there is no place, such as in local-definitions, where an assessment-platform that was used an origin to observe or identify can be place when there is no SSP supplied.  Yet they are referenced via an origin actor by id.  I noticed that they show up in local-definitions in results of an assessment-result model.  I also noticed that in order to populate the weakness detector source column, the assessment-platform would need it as well.

Am I missing something or is this just an oversight as a result of a POA&M without an SSP?

**Comment by aj-stein-nist** [comment]
Thanks for reporting this, this will be resolved in the follow-up to the issue you opened in usnistgov/OSCAL#1291.
